README.md

```diff
@@ -1,18 +1,22 @@
 ﻿# SOEN341-InsertName
 ### Authors
-* Simon Lim (tech lead) | @nasaku898
-* Émilie Martin | @emilie-martin
-* Daniela Venuta | @daniela-venuta
-* Minh-Tam Do | @mint47
-* Luc Nguyen | @1999lucnguyen
-* Karin Kazarian | @Karin-kazarian
+Simon Lim (tech lead) | @nasaku898
+:--|:--
+Émilie Martin         | @emilie-martin  
+Daniela Venuta        | @daniela-venuta
+Minh-Tam Do           | @mint47
+Luc Nguyen            | @1999lucnguyen  
+Karin Kazarian        | @Karin-kazarian  
 
 ### Description
-This is a project partial fulfillement of the requirements for the course SOEN341 at Concordia University.
+This is a social media project based on the main features of instagram. This project is an introduction that we created to help us learn  the basics of industrial software development process.
+The project was created for SOEN341 at Concordia University.
```

Comment:
We might want to write the full name of the class (Software Process)? I think it would be more beneficial to do so, rather than write the class code.